SAMtags.tex

```diff
@@ -90,11 +92,13 @@ \section{Standard tags}
   {\tt PQ} & i & Phred likelihood of the template \\
   {\tt PT} & Z & Read annotations for parts of the padded read sequence \\
   {\tt PU} & Z & Platform unit \\
-  {\tt QT} & Z & Barcode ({\tt BC} or {\tt RT}) phred-scaled base qualities \\
-  {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag \\
+  {\tt QT} & Z & Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}. \\
```

Comment:
Could we be explicit instead of saying "same encoding as QUAL" especially since that's now in a different document.  E.g. `Phred+33 ascii representation of....`